framework/packages/abstract-core/src/objects/account/account_trace.rs

```diff
@@ -34,7 +34,19 @@ impl<'a> PrimaryKey<'a> for AccountTrace {
         match self {
             AccountTrace::Local => LOCAL.key(),
             AccountTrace::Remote(chain_name) => {
-                chain_name.iter().flat_map(|c| c.str_ref().key()).collect()
+                let len = chain_name.len();
```

Comment:
Remote traces are serialized here differently so loading them would be problematic. 

This shouldn't be an issue because there hasn't been the ability to create ICAAs yet.